tests/e2e/check_hydration.js

```
@@ -0,0 +1,80 @@
+    page.on("pageerror", (err) => {
+      pageErrors.push(err.message);
+    });
+
+    await page.goto(url, { waitUntil: "networkidle", timeout: 30000 });
+
+    // Wait for any async hydration warnings to surface
+    await page.waitForTimeout(2000);
+
+    if (pageErrors.length > 0) {
+      console.error("Page errors:");
+      pageErrors.forEach((e) => console.error("  ", e));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)